module1/src/com/epam/linearprograms/HmeTask_1_2.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.linearprograms;
+
+public class HmeTask_1_2 {
+    public static void main(String[] args) {
+        double a = 1.0;
+        double b = 5.0;
+        double c = 2.0;
+        double z;
+        if ((b * b + 4 * a * c < 0) || a == 0 || b == 0){
```

Comment:
change the logical order of check
```suggestion
        if ((b * b + 4 * a * c >= 0) || a != 0 || b != 0){
```